test/errors/tupleDecls-detailed.good

```diff
@@ -0,0 +1,23 @@
+─── error in tupleDecls.chpl:1 [TupleDeclNotTuple] ───
+  Attempt to use tuple declaration to split a value of non-tuple type 'type UnknownType'.
+  In the following tuple declaration:
+      |
+    1 | var (x,y) = notKnown;
+      |
+  the value being assigned has type 'type UnknownType', while it is expected to be a 2-element tuple.
```

Comment:
"the value being assigned" sounds like a description of the LHS to me.  Maybe "the initialization expression has type..."?